### ISSUE TITLE
`gtzan_embeddings.py` needs to be run twice to work

### ISSUE THREAD

**Issue by orrp** [hugofloresgarcia/vampnet#22]
The first time `gtzan_embeddings.py` is run on a dataset, it will error out with
```python
Traceback (most recent call last):
vampnet/scripts/utils/gtzan_embeddings.py", line 263, in <module>
    main()
lib/python3.9/site-packages/argbind/argbind.py", line 159, in cmd_func
    return func(*cmd_args, **kwargs)
vampnet/scripts/utils/gtzan_embeddings.py", line 246, in main
    embeddings = np.stack(embeddings)
  File "<__array_function__ internals>", line 180, in stack
lib/python3.9/site-packages/numpy/core/shape_base.py", line 422, in stack
    raise ValueError('need at least one array to stack')
ValueError: need at least one array to stack
```

Running `gtzan_embeddings.py` again will generate the plots as expected. It is likely an issue with caching.

**Comment by orrp** [comment]
(P.S. the script is awesome, thanks for adding it!)

**Comment by hugofloresgarcia** [comment]
thanks for catching these!! apologies for being slow to reply to issues, have been doing ISMIR stuff the past couple of weeks. I'll hopefully get to fixing these next week :)

**Comment by orrp** [comment]
No probs at allll.. Have fun at ISMIR! None of these are blocking for us, I just figure I'll report them as they come along.

**Comment by orrp** [comment]
Fixed with #23.
